lib/web/sessions.go

```diff
@@ -850,6 +850,19 @@ func (s *sessionCache) invalidateSession(ctx context.Context, sctx *SessionConte
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	if err := clt.DeleteUserAppSessions(ctx, &proto.DeleteUserAppSessionsRequest{Username: sctx.GetUser()}); err != nil {
+		return trace.Wrap(err)
```

Comment:
I realize this is pre-existing, but I wonder how correct it is to abort deletion halfway. Maybe we should do all deletions once this starts, and then return the joined error from all outcomes.

WDYT @flyinghermit and reviewers?

Reply:
IMO that seems like a good move. Otherwise we won't best-effort try to get rid of everything.

Reply:
Yeah I was just being lazy hoping someone else would take that effort. Updated it so the error is combined in https://github.com/gravitational/teleport/pull/37619/commits/8efeb62deca926edc0a9118b926c5c466ba50919
lmk if that's the correct way to do that!

Reply:
Thanks for taking the plunge. I would use an `[]error` for the impl (see comments) but that's kind of minor.